Added a paragraph to the story and some ASCII art

floridop opened the pull request:
Added some content to your story. You are free to ask me to do any change you like before approving!

floridop committed:
Added a paragraph to the story and some ASCII art

udi251nikolei commented on README.md:
```diff
@@ -1,3 +1,32 @@
 
-Once upon a time, in a land closer than you think, there lived a **rich** but _cruel_ king - The Coldran of Kor. He was feared by all his subject and everyone who had the unfortunate chance to hear his name spoken.
+Once upon a time, in a land closer than you think, 
+there lived a **rich** but _cruel_ king - The Coldran of Kor. 
```
I'd like to follow up line 3 with line 4 after the comma.

udi251nikolei commented on README.md:
```diff
@@ -1,3 +1,32 @@
+Once upon a time, in a land closer than you think, 
+there lived a **rich** but _cruel_ king - The Coldran of Kor. 
+He was feared by all his subject and everyone who had the unfortunate 
+chance to hear his name spoken.
```
I don't want line 6 to be a separated from line 5. Follow up line 5 with line 6.

udi251nikolei reviewed (changes requested):
Thanks for your contribution. I saw you have made some edits on my first draft. I've written instructions in the 2 comments I've provided on specific line I want changes.

floridop committed:
Implementing requested changes:

- https://github.com/udi251nikolei/MNXB01-learn/pull/2/files#r995975138
- https://github.com/udi251nikolei/MNXB01-learn/pull/2/files#r995976461

floridop commented:
Requested changes implemented in commit [0c7968b](https://github.com/udi251nikolei/MNXB01-learn/pull/2/commits/0c7968bc7c518222d3a9c9a6bbe699ff2ed4bf25)

floridop commented:
Hello! could you please check that you're fine with the changes and eventually merge?

floridop commented:
Information about merging when you're the release manager is in Tutorial 5 slides 78-82

udi251nikolei commented:
Merging completed

floridop commented:
I'll close this part of the homework, but if there is something else 
you're curious to try and I am available I'll be happy to test.
